### DISCUSSION TITLE
Missing Rows / Errors Messages

### DISCUSSION THREAD

**Discussion by markalanevans** [JoshClose/CsvHelper#2055] [General]
Hi, what happens during  GetRecords call? 
The file i have get's loaded, but the # of rows in the file is always more than what is returned and i'm wondering why. How can i capture this?

**Comment by JoshClose** [comment]
A field may contain a line endings.

For this example

```cs
void Main()
{
    var s = new StringBuilder();
    s.Append("Id,Name\r\n");
    s.Append("1,\"one\r\n\r\n\r\n\r\ntwo\"\r\n");
    var config = new CsvConfiguration(CultureInfo.InvariantCulture)
    {
    };
    using (var reader = new StringReader(s.ToString()))
    using (var csv = new CsvParser(reader, config))
    {
        while (csv.Read())
        {
            csv.RawRecord.Dump();
        }
        
        csv.Row.Dump();
        csv.RawRow.Dump();
    }
}
```
![image](https://user-images.githubusercontent.com/114971/195872506-7ca8d1ff-a827-42a7-a68c-551005bf7fad.png)

You can see that there are `3` CSV rows, but `7` rows in a text editor.
